metricbeat/module/docker/diskio/data.go

```diff
@@ -0,0 +1,30 @@
+package diskio
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+)
+
+func eventsMapping(blkioStatsList []BlkioStats) []common.MapStr {
+	myEvents := []common.MapStr{}
+	for _, blkioStats := range blkioStatsList {
+		myEvents = append(myEvents, eventMapping(&blkioStats))
+	}
+	return myEvents
+}
+func eventMapping(myBlkioStats *BlkioStats) common.MapStr {
+	event := common.MapStr{
+		"@timestamp": myBlkioStats.Time,
```

Comment:
What kind of timestamp is that? I would only use one `@timestamp` which is generated automatically. In case this timestamp is no `now()` I recommend to call it `timestamp` or a name that fits better.